app/bundles/PointBundle/Views/Trigger/form.html.php

```diff
@@ -40,6 +40,7 @@
                                     <?php
                                     echo $view['form']->row($form['name']);
                                     echo $view['form']->row($form['description'], ['attr' => ['class' => 'form-control editor']]);
+                                    echo $view['form']->row($form['scoringCategory']);
```

Comment:
let's re-arrange this UI a bit  ![2017-10-06_18-29-53](https://user-images.githubusercontent.com/63312/31302065-871f987c-aac4-11e7-9d17-a2429a3c207a.png)

Reply:
Don't know how to do that, maybe it is a 5 minute topic ?

In this case, could you maybe finish to polish the UI the way you expected to have it ?


We spent 200 hours of R&D on the topic, and took time to design specs available to community before the development (see PDF upper page 6). We have customers already using the module the way we pushed it to github, and they love using it ! It is hard for us to be able to change all the behavior or functionnalities.